src/cli/test_shared_memory_system.zig

```diff
@@ -142,7 +142,7 @@ test "integration - shared memory setup and parsing" {
     try testing.expect(shm_handle.size > 0);
     try testing.expect(@intFromPtr(shm_handle.ptr) != 0);
 
-    std.log.info("Integration test: Successfully set up shared memory with size: {} bytes\n", .{shm_handle.size});
+    std.log.debug("Integration test: Successfully set up shared memory with size: {} bytes\n", .{shm_handle.size});
```

Comment:
Switch to log.debug, these aren't printed in ReleaseSafe builds but they are in dev/debug builds which is helpful to see.